xml/System.Reflection/ProcessorArchitecture.xml

```diff
@@ -61,7 +61,7 @@
         <ReturnType>System.Reflection.ProcessorArchitecture</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>A 64-bit AMD processor only.</summary>
+        <summary>A 64-bit processor based on the x86-64 architecture.</summary>
```

Comment:
The preferred wording is x64 architecture. You could also do this as a link: \[x64 architecture]\(/windows-hardware/drivers/debugger/x64-architecture).